client/src/components/Results.jsx

```diff
@@ -46,11 +49,24 @@ class Results extends React.Component {
     });
   }
 
+  handleFavColleges(college) {
```

Comment:
Let's try and do 2 big things with this:

1) Let's write it so it toggles fav colleges on multiple clicks rather than adding the college over and over.
2) Let's see if we can make it generic and pass it to FavoritesPage.jsx OR let's write a very similar function in there that adjusted to the state in FavoritesPage.jsx and passed to ResultsListEntry.jsx .  This would permit us to remove favorites from the favorites page.